android/app/src/main/assets/event

```diff
@@ -78,7 +78,8 @@
         "starts-at": "2016-11-01T16:00:00+08:00",
         "privacy": "public",
         "ends-at": "2017-01-20T17:00:00+08:00",
-        "hash": ""
+        "hash": "",
+        "email-subject": "Be a Speaker at the FOSSASIA Summit"
```

Comment:
Please don't change this file. This is a copy of the JSON response so that the app can work offline. These changes would only work when the app uses this JSON file and not when it requests it from the server.

Reply:
@nikit19 I will revert this :+1: